android/src/main/java/org/ligi/passandroid/model/pass/PassImpl.kt

```diff
@@ -61,6 +60,11 @@ class PassImpl(override val id: String) : Pass {
 
     override var passIdent: String? = null
 
+    override fun isExpired(): Boolean {
+        val to = validTimespans.get(0).to
```

Comment:
when a pass has no valid-timespan this would crash - should better return false in this case